java/dev/checkstyle/checkstyle.xml

```diff
@@ -48,9 +48,7 @@
       <property name="file" value="${checkstyle.suppressions.file}"/>
     </module>
 
-    <module name="NewlineAtEndOfFile">
-        <property name="lineSeparator" value="lf" />
```

Comment:
Why are we removing the property here? Wouldn't that confusion for people using Windows line separator?

Reply:
Good call, I should fix this properly.

The windows build was always generating windows line separators. There was a bug in the old version of checkstyle that couldn't differentiate between \lf and \cr\lf, so this was never caught before. https://github.com/checkstyle/checkstyle/issues/4074

Reply:
Fixed! I just had to update `java/.gitattributes`